components/organisms/TitleSection.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+/**
+ *  page title section
+ */
+export default function TitleSection(props) {
+  return (
+    <div>
+      <h2 className="pb-2 font-bold">{props.title}</h2>
+      <hr className="border-red-800 border-2"></hr>
```

Comment:
Alpha site has a color defined in the tailwind config called hr-red-bar, which is a slightly lighter shade and looks closer to the Figma design.

Instead of border-red-800, we should use border-hr-red-bar here

Reply:
Upon further testing, rather than use border, the horizontal rule should be as follows:
```<hr className="h-1 bg-hr-red-bar"></hr>```

Reason being is that when you scroll in and out you can see the outline of the border, with the above snippet that isn't an issue

Reply:
good call!  i was curious if there was something sensible to colour this